### DISCUSSION TITLE
Twain Protocol / Scanning Support

### DISCUSSION THREAD

**Discussion by MrZakaz** [ImageMagick/ImageMagick#3391] [General]
Hello, I would like to know if imagemagick has the Twain protocol or some scanning support - And if not what the authors suggest it supports?
What can I do for my SkanPort with ImageMagic to be able to download an image from the scanner?

**Comment by snibgo** [comment]
See the "SCAN" and "SCANX" formats on https://www.imagemagick.org/script/formats.php

What platform do you use? "SANE" doesn't seem to be available on Windows.

**Reply by MrZakaz** [reply]
I'm using windows magick.net
